python/lsst/phosim/utils/phosim_repackager.py

```diff
@@ -53,11 +53,13 @@ def process_visit(self, visit_dir, out_dir=None, verbose=False):
         out_dir: str [None]
             Output directory for MEF files. If None, then a directory
             with name v<visit #>-<band> will be created in the cwd.
+        prefix: str ['lsst']
+            Prefix to use to find PhoSim amplifier files in visit_dir.
         verbose: bool [False]
             Set to True to print out time for processing each sensor.
         """
         phosim_amp_files \
-            = sorted(glob.glob(os.path.join(visit_dir, 'lsst_a_*')))
+            = sorted(glob.glob(os.path.join(visit_dir, '{}_a_*'.format(prefix))))
```

Comment:
I'm in favor of using f-strings instead of `.format`.  This would mean changing `'{}_a_*'.format(prefix)` to `'{prefix}_a_*'`.

I like them because they are really compact.

Reply:
Nice, will change. Thank you for introducing me to f-strings.

Reply:
`f'{prefix}_a_*'` to be clear.